src/main/java/io/zeebe/containers/cluster/ZeebeClusterBuilder.java

```diff
@@ -94,6 +96,7 @@
   private static final String BROKER_NETWORK_ALIAS_PREFIX = "zeebe-broker-";
   private static final String GATEWAY_NETWORK_ALIAS_PREFIX = "zeebe-gateway-";
   private static final String DEFAULT_CLUSTER_NAME = "zeebe-cluster";
+  private static final ZeebeDefaults ZEEBE_DEFAULTS = ZeebeDefaults.getInstance();
```

Comment:
Yep, I am thinking there to avoid duplications with the prefix `ZeebeDefaults.`. But I think that's the better approach is just to import the static method.

Reply:
Done